### DISCUSSION TITLE
Hi dear sionna developers, Ques abt. the Ray Tracing Module.

### DISCUSSION THREAD

**Discussion by kaimo455** [NVlabs/sionna#53] [Q&A] [answered]
Hi dear `sionna` developers,

I was searching around about __*some APIs or codes about the RayTracing-related module*__ in `sionna`, cause I am currently doing researches in this area. Well cause as we know, the RTX series graphic cards support RayTracing, so I was wondering does this repo provide some easy-to-use APIs to call so that I can propagate rays in a 3D map (raster or vector map) from multiple out points to entire map space?

**Comment by jhoydis** [comment, marked as answer]
Sionna does not support ray tracing for the moment. We are actively working on this at the moment.

**Reply by kaimo455** [reply]
Thanks @jhoydis 

btw: that was a really instant reply :)
